### ISSUE TITLE
模型集成到Flask框架中去预测时候，每次启动项目报错。No stack trace in paddle, may be caused by external reasons.

### ISSUE THREAD

**Issue by suntao2015005848** [PaddlePaddle/PaddleTS#453]
docker环境中运行：
paddlets：1.1.0
paddlepaddle: 2.4.0
aiosignal            1.3.1
alembic              1.11.2
aniso8601            9.0.1
APScheduler          3.10.4
astor                0.8.1
attrs                23.1.0
backcall             0.2.0
backports.zoneinfo   0.2.1
cachelib             0.9.0
certifi              2023.7.22
charset-normalizer   3.2.0
chinese-calendar     1.8.0
click                8.0.4
cloudpickle          2.2.1
cmaes                0.10.0
colorama             0.4.6
colorlog             6.7.0
ConfigSpace          0.7.1
cycler               0.11.0
DBUtils              3.0.3
decorator            5.1.1
distlib              0.3.7
filelock             3.12.2
FLAML                1.2.4
Flask                2.2.5
Flask-Caching        2.0.2
Flask-Cors           4.0.0
flask-restx          1.1.0
fonttools            4.38.0
frozenlist           1.3.3
greenlet             2.0.2
grpcio               1.43.0
hpbandster           0.7.4
idna                 3.4
importlib-metadata   6.7.0
importlib-resources  5.12.0
ipython              7.34.0
itsdangerous         2.1.2
JayDeBeApi           1.2.3
jedi                 0.19.0
Jinja2               3.1.2
joblib               1.3.2
JPype1               1.4.1
jsonschema           4.17.3
kiwisolver           1.4.4
lightgbm             4.0.0
llvmlite             0.39.1
lxml                 4.9.3
Mako                 1.2.4
MarkupSafe           2.1.3
matplotlib           3.5.3
matplotlib-inline    0.1.6
more-itertools       9.1.0
msgpack              1.0.5
netifaces            0.11.0
numba                0.56.4
numpy                1.19.5
opt-einsum           3.3.0
optuna               3.3.0
packaging            23.1
paddle-bfloat        0.1.7
pandas               1.3.5
parso                0.8.3
patsy                0.5.3
pexpect              4.8.0
pickleshare          0.7.5
Pillow               9.5.0
pip                  23.2.1
pkgutil_resolve_name 1.3.10
platformdirs         3.10.0
prompt-toolkit       3.0.39
protobuf             3.20.0
ptyprocess           0.7.0
Pygments             2.16.1
pyod                 1.1.0
pyparsing            3.1.1
Pyro4                4.82
pyrsistent           0.19.3
python-dateutil      2.8.2
python-docx          0.8.11
pytz                 2023.3
PyWavelets           1.3.0
PyYAML               6.0.1
ray                  1.13.0
requests             2.31.0
scikit-learn         1.0.2
scipy                1.7.3
seaborn              0.12.2
serpent              1.41
setuptools           41.6.0
shap                 0.42.1
six                  1.16.0
slicer               0.0.7
SQLAlchemy           2.0.19
statsmodels          0.12.2
tabulate             0.9.0
tensorboardX         2.6.2
threadpoolctl        3.1.0
tqdm                 4.66.1
traitlets            5.9.0
typing_extensions    4.7.1
tzdata               2023.3
tzlocal              5.0.1
urllib3              2.0.4
virtualenv           20.24.2
wcwidth              0.2.6
Werkzeug             2.2.3
wheel                0.33.6
xgboost              1.6.2
zipp                 3.15.0
![image](https://github.com/PaddlePaddle/PaddleTS/assets/49387197/0c89d847-1db4-4d02-869d-6b6514368f00)
报错后，会在项目根目录下出现一个core文件，大约占用2.3g。
![image](https://github.com/PaddlePaddle/PaddleTS/assets/49387197/07309506-ea81-4311-93a9-8d018b0833ea)

**Comment by Sunting78** [comment]
是需要进行服务化部署吗？如果本地跑会报错吗？

**Comment by Sunting78** [comment]
您好，超过一个月该issue已关闭，如有疑问可以reopen或新提issue~
